versions/3.0.md

```diff
@@ -2974,10 +2974,13 @@ Supported schemes are basic authentication, an API key (either as a header or as
 ##### Fixed Fields
 Field Name | Type | Validity | Description
 ---|:---:|---|---
-<a name="securitySchemeType"></a>type | `string` | Any | **Required.** The type of the security scheme. Valid values are `"basic"`, `"apiKey"` or `"oauth2"`.
+<a name="securitySchemeType"></a>type | `string` | Any | **Required.** The type of the security scheme. Valid values are `"apiKey"`, `"http"`, `"oauth2"`, `"openIdConnect"`.
```

Comment:
I'm going to assume `openIdConnect` you mean http://openid.net/connect/ ?  Is that a generic protocol?  Is it ok to use the name in the spec? 

I'm wondering if there is a more generic term to describe it.